asyncapi/asyncapi.yml

```diff
@@ -251,33 +250,31 @@ channels:
             retain: true
   pe/dpi_ack:
     description:
-      $ref: markdown/dpi-acknowledge.md
+      $ref: json-schemas/pe/dpi_ack/dpi-acknowledge.md
     publish:
       message:
-        name: DPIAcknowledge Message
+        name: DpiAcknowledge
         schemaFormat: application/schema+json;version=draft-07
         payload:
-          $ref: json-schemas/dpi-acknowledge.json
-        examples: [
-          { $ref: examples/dpi-acknowledge.json }
-        ]
+          $ref: json-schemas/pe/dpi_ack/dpi-acknowledge.json
+        examples:
+          - $ref: json-schemas/pe/dpi_ack/dpi-acknowledge.example.json
         bindings:
           mqtt:
             qos: 1
             retain: false
   pe/dpi_diagnostics:
```

Comment:
@eivindga should we change this to `pe/dpi/diagnostics`?

Reply:
yes